sample/src/main/java/com/polidea/cockpit/sample/apipresentation/ApiPresentationBaseContract.kt

```diff
@@ -0,0 +1,24 @@
+package com.polidea.cockpit.sample.apipresentation
+
+import android.support.annotation.ColorInt
+import com.polidea.cockpit.sample.BasePresenter
+import com.polidea.cockpit.sample.BaseView
+
+interface ApiPresentationBaseContract {
+
+    interface View<T : Presenter> : BaseView<T> {
+        fun showFooter(isVisible: Boolean)
+
+        fun setFooterText(footerText: String)
+
+        fun setTextColor(@ColorInt color: Int)
+
+        fun setFontSize(textSize: Float)
+
+        fun setColorDescription(description: String)
+    }
+
+    interface Presenter : BasePresenter {
+        fun initViews()
```

Comment:
`initViews()` method is never called in `ApiPresentationBaseContract.View`. The method is called in `ApiPresentationBasePresenter.start()` method only. If the method is not called in View, it should be not exposed to `Presenter` interface. The other way is to move `initViews()` call from `ApiPresentationBasePresenter.start()` to `ApiPresentationBaseFragment.onViewCreated()` or similar. It's up to you.